packages/traceloop-sdk/traceloop/sdk/tracing/tracing.py

```
@@ -428,7 +428,8 @@ def init_instrumentations(
     block_instruments: Optional[Set[Instruments]] = None,
 ):
     block_instruments = block_instruments or set()
-    instruments = instruments or set(
+    # explictly test for None since empty set is a False value
+    instruments = (instruments == None) or set(
```

Ruff (0.12.2)

Comparison to `None` should be `cond is None`

Replace with `cond is None`

(E711)